Widgets/Animation/VarmintWidgetAnimationCollection.cs

```diff
@@ -4,13 +4,29 @@
 
 namespace MonoVarmint.Widgets
 {
-    //--------------------------------------------------------------------------------------
-    /// <summary>
-    /// MoveOffsetLinear - animate an offset from one value to another
-    /// </summary>
-    //--------------------------------------------------------------------------------------
     public partial class VarmintWidgetAnimation
     {
+        //--------------------------------------------------------------------------------------
+        /// <summary>
+        /// CycleFade - Fades between a start and end opacity in a sinusoidal manner
+        /// </summary>
+        //--------------------------------------------------------------------------------------
+        public static VarmintWidgetAnimation CycleFade(
+            double durationSeconds,
+            float startOpacity,
+            float endOpacity,
+            float cycleTime
+            )
+        {
+            var opacityChange = startOpacity - endOpacity;
+            return new VarmintWidgetAnimation(durationSeconds, (widget, delta) =>
+            {
+                var amplitude = (startOpacity - endOpacity) / 2;
```

Comment:
Simpler to say opacitychange/2.  This can also be defined outside of the lamba, so we don't have to recalculate it.